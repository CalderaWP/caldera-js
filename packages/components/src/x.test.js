import React from 'react';
import renderer from 'react-test-renderer';
import {mount} from 'enzyme';

import {A1} from './index';
describe( 'a', () => {
	it( 'snapshots', () => {
		expect(renderer.create( <A1 />).toJSON()).toMatchSnapshot();
	})
})
