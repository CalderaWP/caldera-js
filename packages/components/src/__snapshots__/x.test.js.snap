// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`a snapshots 1`] = `
<div>
  a1
</div>
`;
